config/projects/supermarket.rb

```diff
@@ -32,7 +32,7 @@
 override :bundler, version: "1.7.3"
 override :'chef-gem', version: "11.16.4"
 override :postgresql, version: "9.3.4"
-override :'omnibus-ctl', version: "nls/ignore-config"
+override :'omnibus-ctl', version: "master"
```

Comment:
I just tagged an 0.3.3 that includes this change, and opened chef/omnibus-software#372. We could `bundle update omnibus-software` once that's merged and be able to remove this line altogether.